backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1285,7 +1285,8 @@ class WorkspaceDigestSchema(marshmallow.Schema):
     label = StrippedString(example="Intranet")
 
 
-class WorkspaceSchema(WorkspaceDigestSchema):
+# NOTE - SG - 2021-04-29 - Used to avoid transmitting description in all TLMs
+class WorkspaceWithoutDescriptionSchema(WorkspaceDigestSchema):
```

Comment:
Just to understood, what the difference between "WorkspaceWithoutDescriptionSchema" and  a "DigestWorkspaceSchema" (does this one exist ?).